### PR TITLE
Strings reworked

### PR DESCRIPTION
**Pull request by comradekingu**

### CODE REVIEW

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -317,20 +317,20 @@
     <string name="account_about_subscribers">Subscribers:</string>
     <string name="account_about_description">Description:</string>
     <string name="account_about_joined">Joined:</string>
-    <string name="api_error">Something went wrong, please try later!</string>
+    <string name="api_error">Something went wrong, please try later.</string>
     <string name="action_bar_title_server_selection">Select Server</string>
 
     <string name="hello_blank_fragment">Hello blank fragment</string>
     <string name="action_bar_title_address_book" />
```
Comment (comradekingu) [inline app/src/main/res/values/strings.xml:324]: Empty string?

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -317,20 +317,20 @@
...
-    <string name="authentication_login_success">Login Successful</string>
-    <string name="authentication_login_failed">Login Failed!</string>
-    <string name="server_book_no_servers_found">Server Books is empty</string>
```
Comment (comradekingu) [inline app/src/main/res/values/strings.xml:327]: Not sure what "Server Books" is.

Comment (Poslovitch) [reply app/src/main/res/values/strings.xml:327]: Server book is like an "address book", somewhere you list the instances you want to connect to.

**Comment by sschueller** [comment]
Thank you, may I ask you to make these changes in weblate at https://hosted.weblate.org/projects/peertube/android/en/ ?

**Pull request closed without merging**